test_pull_requests

```diff
@@ -99,14 +99,18 @@ config = args[:config]
 merge_pretest_success = args[:merge_pretest_success]
 dry_run = args[:dry_run]
 
-PROPERTIES_LOCATION = File.expand_path(config || DEFAULT_PROPERTIES_LOCATION)
-Properties = if File.exists? PROPERTIES_LOCATION
-  JSON.parse(IO.read(PROPERTIES_LOCATION))
-else
-  $stderr.puts "Properties file not found: #{PROPERTIES_LOCATION}"
-  exit 255
+def load_properties
+  if File.exists? PROPERTIES_LOCATION
```

Comment:
This lets us make the bailout code conditional, and makes the bit on line 112 a little easier to read.